modules/ROOT/pages/7.2-release-notes.adoc

```diff
@@ -127,6 +127,17 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 Previously in {productname}, the `referrerpolicy` attribute was not considered as valid for iframe elements. In {productname} {release-version}, this attribute has been added to the schema, allowing users to include the `referrerpolicy` attribute in iframes.
 
+=== Improved Search & Replace dialog accessibility by changing placeholders to labels.
+// #TINY-10871
+
+In previous versions of {productname}, the Find & Replace dialog inputs were initially implemented with placeholders.
+
+As a consequence, the dialog was less accessible because the placeholders are removed once the user starts typing. This posed difficulties for people with cognitive disabilities who rely on constant visual cues for context.
+
+{productname} {release-version} addresses this issue. Now, placeholders have been replaced by input labels.
+
+As a result, dialog is now more accessible and the labels remain visible at all times.
```

Comment:
```suggestion
In previous versions of {productname}, the Search & Replace dialog inputs were initially implemented with placeholders.

As a consequence, the dialog was less accessible because the placeholders are removed once the user starts typing. This posed difficulties for people with cognitive disabilities who rely on constant visual cues for context.

{productname} {release-version} addresses this issue. Now, placeholders have been replaced by input labels.

As a result, dialog is now more accessible and the labels remain visible at all times.
```